bot/cogs/verification.py

```diff
@@ -65,10 +54,7 @@ async def on_message(self, message: Message) -> None:
 
         if message.author.bot:
             # They're a bot, delete their message after the delay.
-            # But not the periodic ping; we like that one.
-            if message.content != PERIODIC_PING:
-                await message.delete(delay=BOT_MESSAGE_DELETE_DELAY)
-            return
```

Comment:
Is the deletion of the `return` statement intentional?

Reply:
No, and it's also a bad idea. Nice catch!